deploy/runtime/ui/Battery Dispatch Options FOM.txt

```diff
@@ -476,6 +480,7 @@ function warning_message()
 	{
 		msg = ''; 
 	}
+	/*
 	elseif ( value('batt_dispatch_choice_ui') == 3 ) 
 	{ 
 		if ( technology() != 'PV Battery' )
```

Comment:
So is the final decision to give a warning to the user that the PVSmoothing option is only intended for PV Battery configs or not?